src/Sulu/Component/Rest/ListBuilder/Metadata/Resources/schema/list-2.0.xsd

```diff
@@ -72,18 +72,20 @@
         <xs:attributeGroup ref="fieldAttributeGroup"/>
     </xs:complexType>
 
-    <xs:complexType name="parametersType">
+    <xs:complexType name="filterType">
         <xs:sequence>
-            <xs:element type="parameterType" name="parameter" minOccurs="1" maxOccurs="unbounded"/>
+            <xs:element type="paramType" name="param" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
inf the form xml files, we use a `<params>` tag that wraps the parameters? should we use that here too? 

maybe the filters will get a `<meta>` tag too in the future? eg for defining a filter name?

Reply:
I already hat the `params` tag there :see_no_evil: But if the majority wants to have them I am fine with introducing it.

I don't see any reason `meta` tags should not be able to be added later, so I would not do that right now.

Reply:
yes the `meta` tag can be added later. i just figured that this might be the reason for the `params` tag in the form xmls. not sure if i like the `params` tag there, but i think we should keep things consistent? 🤔

Reply:
Ok, consistency argument wins, will add it.